src/main/java/org/opentripplanner/routing/api/request/preference/RaptorPreferences.java

```diff
@@ -31,18 +34,27 @@ public final class RaptorPreferences implements Serializable {
 
   private final Instant timeLimit;
 
+  private final double relaxTransitSearchCostCriteria;
+
   private RaptorPreferences() {
     this.optimizations = EnumSet.of(Optimization.PARETO_CHECK_AGAINST_DESTINATION);
     this.profile = RaptorProfile.MULTI_CRITERIA;
     this.searchDirection = SearchDirection.FORWARD;
     this.timeLimit = null;
+    this.relaxTransitSearchCostCriteria = SearchParams.NOT_SET;
```

Comment:
I think the name should include `AtDestination`. It is better to use `GeneralizedCost` than `CostCriteria` to be consistent. So `relaxTransitSearchGeneralizedCostAtDestination` ?